__tests__/istanbul-linker-util.ts

```diff
@@ -136,14 +142,14 @@ class Recurse {
   }
 }
 
-module.exports = function (fn) {
+module.exports = function (fn: Object) {
```

Comment:
`Object` is a generic, we may want a `function` definition here.

Reply:
done